tests/_support/ea-server-mocker/ea-service-mocker.php

```diff
@@ -0,0 +1,118 @@
+<?php
+/**
+ * Plugin Name: Event Aggregator Server Mocker
+ * Plugin URI: http://theAverageDev.com
+ * Description: Mock Event Aggregator Server responses and interactions; symlink this from its location to the WordPress plugins
+ * folder. Version: 1.0 Author: Modern Tribe Author URI: http://theeventscalendar.com License: Private
+ */
+
+$mocker = new Tribe__Events__Aggregator_Mocker();
+
+spl_autoload_register( array( $mocker, 'autoload' ) );
+
+add_action( 'plugins_loaded', array( $mocker, 'mock' ), -1 );
+
+register_deactivation_hook( __FILE__, array( 'Tribe__Events__Aggregator_Mocker', 'deactivate' ) );
+
+require_once dirname( __FILE__ ) . '/src/functions.php';
+
+class Tribe__Events__Aggregator_Mocker {
+
+	/**
+	 * Adding a class that uses  options? Add it here.
+	 *
+	 * Keep in mind this is a testing tool and you want to put your nose in
+	 * the database to check for stuff; refrain from using serialized options
+	 * and make your life easier: you do not want to look at nested serialized arrays.
+	 * Your database will survive the burden of the additional options: don't worry.
+	 *
+	 * @var Tribe__Events__Aggregator_Mocker__Option_Provider_Interface[]
+	 */
+	protected static $option_providers
```

Comment:
How about:

```php
protected static $option_providers = array(
  'Tribe__Events__Aggregator_Mocker__Service_Options',
);
```

It'll more closely match other multi-line standards we have and will save on one level of tabbing. Rinse and repeat this comment for other places this occurs in the plugin.